manifest.json

```diff
@@ -38,9 +40,5 @@
           }
     },
     "default_locale": "en",
-    "manifest_version": 2,
-    "-ms-preload": {
-        "backgroundScript": "js/ms/backgroundScriptsAPIBridge.js",
-        "contentScript": "js/ms/contentScriptsAPIBridge.js"
-    }
+    "manifest_version": 2
```

Comment:
Does this need to be bumped? Does it matter?

Reply:
No, that's the version of the style of manifest.